src/grpc_generated/go/README.rst

```diff
@@ -27,31 +27,32 @@
 
 |License|
 
+Regenerate Go Client stubs
```

Comment:
```suggestion
Generate Go Client stubs
```

Reply:
Regenerate may be the better word choice here. The stubs are already in the repo, so we're regenerating/refreshing them. Perhaps there's a clearer way to convey that?